spec/libraries/filename_sanitizer_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe FilenameSanitizer do
+  let(:test_class) { Class.new { include FilenameSanitizer } }
+  describe 'sanitize' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.